sha3/src/main/java/com/theromus/exception/MustBeSetTypeHashException.java

```diff
@@ -0,0 +1,3 @@
+package com.theromus.exception;
+@SuppressWarnings("serial")
+public class MustBeSetTypeHashException extends RuntimeException {}
```

Comment:
Remove this class. Let's use one Exception - NotValidHashLenException

Reply:
This was to have a more readable error, but I can add a message in the Exception - NotValidHashLenException.